front/src/modules/pathfinding/components/Pathfinding/TypeAndPathV2.tsx

```diff
@@ -224,75 +226,79 @@ const TypeAndPathV2 = () => {
   }, []);
 
```

Comment:
It can be more readable to make early check, insteadd wrap all the component return with `isVisible`. What do you think ?
```suggestion
if (!isVisible) {
    return null;
  }

```

Reply:
Indeed, the effect is the same and it's easier to read.
Thank's ;-)